cmd/agent/senders/http_sender.go

```diff
@@ -32,17 +34,44 @@ func (s *HTTPSender) Send(metrics map[string]interface{}) error {
 		}
 
 		url := fmt.Sprintf("%s/update/%s/%s/%v", s.ServerURL, metricType, key, value)
-		req, err := http.NewRequest(http.MethodPost, url, bytes.NewBuffer(nil))
+
+		// Сжатие тела запроса (в данном случае тело пустое, но это пригодится для реальных данных)
+		var compressedBody bytes.Buffer
+		gzipWriter := gzip.NewWriter(&compressedBody)
+		_, err := gzipWriter.Write([]byte{}) // Пустое тело для текущего примера
+		if err != nil {
+			return err
+		}
+		gzipWriter.Close()
```

Comment:
[CHANGE REQUEST] Please review the Go layouts again.
https://github.com/golang-standards/project-layout/blob/master/cmd/README.md